packages/edit-site/src/components/sidebar/global-styles-v2/components/index.js

```diff
@@ -0,0 +1,8 @@
+export { ColorPickerModal } from './color-picker-modal';
+export { Inspector } from './inspector';
+export { NavLink } from './nav-link';
+export { Screen } from './screen';
```

Comment:
Which of these make sense to be in `@wordpress/components` instead of here?

Reply:
The color picker modal. Not sure what the inspector entails. Screen header would also make sense depending on the design.